docker/entry.sh

```diff
@@ -27,6 +27,14 @@ if ! [ -z "$GEOIPUPDATE_DB_DIR" ]; then
   database_dir=$GEOIPUPDATE_DB_DIR
 fi
 
+if [ ! -z "$GEOIPUPDATE_ACCOUNT_ID_FILE" ]; then
+  GEOIP_UPDATE_ACCOUNT_ID=$( cat $GEOIPUPDATE_ACCOUNT_ID_FILE )
```

Comment:
It would be good to quote the filenames passed to `cat`

Reply:
Done, thanks for picking that up! :)